cmd/policy-admission/main.go

```diff
@@ -95,6 +95,23 @@ func main() {
 				EnvVar: "CONTROLLER_NAME",
 				Value:  "policy-admission.acp.homeoffice.gov.uk",
 			},
+			cli.StringFlag{
+				Name:   "admission-service-name",
+				Usage:  "kuberneres service name of the admission controller, used when registering `SERVICE`",
+				EnvVar: "ADMISSION_SERVICE_NAME",
+				Value:  "policy-admission",
+			},
+			cli.StringFlag{
+				Name:   "failure-policy",
+				Usage:  "the failure policy when registering the controler `POLICY`",
+				EnvVar: "FAILURE_POLICY",
+				Value:  "Ignore",
+			},
+			cli.StringFlag{
+				Name:   "certificate-bundle",
+				Usage:  "file container the certificate authority bundle used when registering `PATH`",
```

Comment:
`file container the certificate authority bundle` => `file containing the certificate authority bundle`